src/site/_includes/js/eligibility.js

```diff
@@ -1489,25 +1489,35 @@ function gaResult(input) {
   const NUM_OF_DEPENDENTS = 0;     // None
   const MAX_RESOURCES = 500;       // USD Combined household assets
 
+  const meetsAgeReq = ge(input.age, MIN_GA_ELIGIBLE_AGE);
+
   const numDependents = input.householdDependents.filter(d => d).length;
+  const hasNoDependents = eq(numDependents, NUM_OF_DEPENDENTS);
 
-  const eligible = and(
-      ge(input.age, MIN_GA_ELIGIBLE_AGE),
-      eq(numDependents, NUM_OF_DEPENDENTS),
-      le(totalResources(input), MAX_RESOURCES),
-      le(grossIncome(input), grossLimit.getLimit(input.householdSize)),
-      or(
-        not(input.notCitizen),
-        isOneOf(input.immigrationStatus, [
-          'permanent_resident',
-          'qualified_noncitizen_gt5y',
-          'qualified_noncitizen_le5y',
-        ])));
+  const underResourceLimit = le(totalResources(input), MAX_RESOURCES);
+  const incomeLimit = grossLimit.getLimit(input.householdSize);
+  const underIncomeLimit = le(grossIncome(input), incomeLimit);
+
+  const meetsImmigrationReq = or(
+    not(input.notCitizen),
+    isOneOf(input.immigrationStatus, [
+      'permanent_resident',
+      'qualified_noncitizen_gt5y',
+      'qualified_noncitizen_le5y',
+    ]));
 
   const program = new Program();
-  // TODO: Replace this single example condition with a set of simplified
-  // conditions describing the separate eligibility requirements.
-  program.addCondition(new EligCondition('Example', eligible));
+  program.addCondition(
+    new EligCondition(`Age ${MIN_GA_ELIGIBLE_AGE} or older`, meetsAgeReq));
+  program.addCondition(
+    new EligCondition(`Number of dependents are ${NUM_OF_DEPENDENTS}`, hasNoDependents));
```

Comment:
What do you think of this alternate wording?

`` `Has ${NUM_OF_DEPENDENTS} dependent children` ``

Reply:
I remember this from sometime ago while doing taxes.
I found a link about this:
https://turbotax.intuit.com/tax-tips/family/rules-for-claiming-a-dependent-on-your-tax-return/L8LODbx94
 It says:
' Typically, this includes your children or other relatives, but it can also include people who aren’t directly related to you, such as a domestic partner.'

Reply:
Ah yes!  I remember doing similar research and landed on a tax website because that's the only other spot I'd ever heard the term "dependent".  Confusingly, different programs have different definitions of "dependent".

Seeing as the [main page](https://socialservices.sccgov.org/financial-assistance/financial-assistance-adults-without-dependent-children-general-assistance) for GA says "without dependent children" I think we're safe using that language on our site.

Reply:
Is the definition consistent across other public assistance programs?

Reply:
The only other program we screen for that uses the "dependent" checkbox is VA Pension.  In addition to dependent children, VA Pension has such a thing as a "dependent spouse", so we have a separate radio button to notate which family member is your spouse.

Reply:
OK